web/src/main/java/org/springframework/security/web/savedrequest/HttpSessionRequestCache.java

```diff
@@ -129,4 +131,15 @@ public void setCreateSessionAllowed(boolean createSessionAllowed) {
 	public void setPortResolver(PortResolver portResolver) {
 		this.portResolver = portResolver;
 	}
+
+	/**
+	 * If the {@code sessionAttrName} property is set, the request is stored in
+	 * the session using this attribute name. Default is
+	 * "SPRING_SECURITY_SAVED_REQUEST".
```

Comment:
@stonio the static constant is package scope, so we will stick with the String in the Javadoc